### PR TITLE
add: Snackbar Component

### PR DESCRIPTION
**Pull request by shilpe26**
Made Snackbar Component with feature:-
🔹Simple button clickable and open a snack bar at the bottom of page

### CODE REVIEW

File: components/snackbar.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    <a class="aside-link" href="./toast.html">Toast</a>
+                </li>
+            </ul>
+        </aside>
+
+
+        <!-- main-section -->
+        <main class="avatar-main">
+            <h1 class="avatar-heading">SnackBar</h1>
+            <!-- <p class="avatar-text">
```
Comment (junaidshaikh-js) [inline components/snackbar.html:88]: We can remove code comment if possible.

**Review by junaidshaikh-js — COMMENTED** [review]
Awesome. Loved the clean code.

File: components/snackbar.html (hunk 1 above)
Comment (shilpe26) [reply components/snackbar.html:88]: yes, I will do but after two more components.

File: css/component_styles/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+.container button {
+    /* color: var(--btn-color); */
+    background-color: transparent;
+    border: none;
+    margin-right: 1rem;
+    font-weight: bold;
+}
+
+.exit i {
```
Comment (anuj-kumary) [inline css/component_styles/snackbar.css:33]: Don't target or give CSS property directly to any Html element try to class and use it.

Comment (shilpe26) [reply css/component_styles/snackbar.css:33]: Done

Comment (shilpe26) [reply css/component_styles/snackbar.css:33]: Done👍🏻

**Pull request merged**